charts/istio-authorizer/values.yaml

```diff
@@ -0,0 +1,81 @@
+## istio-authorizer image parameters
+##
+image:
+  ## Image repository
+  ##
+  repository: acp.artifactory.cloudentity.com/istio-authorizer
+
+  ## Image pull policy
+  ##
+  pullPolicy: IfNotPresent
+
+  ## Image tag (immutable tags are recommended, default is the chart appVersion)
+  ##
+  tag: ""
+
+## istio-authorizer image registry secret names as an array
+##
+imagePullSecrets:
+  - name: artifactory
+
+## String to partially override istio-authorizer.name
```

Comment:
Can we add additional information on what "partially" means?

Reply:
This is low level option. I removed it.